simulation.cpp

```diff
@@ -386,8 +395,8 @@ void test_simulation() noexcept//!OCLINT test may be many
         change_nth_ind_net(s, worst_ind, worst_net);
         auto resp_worst = response(get_nth_ind(s, worst_ind))[0];
 
-        assert(resp_best == get_current_env_value(s));
-        assert(resp_worst != get_current_env_value(s));
+        assert(resp_best == optima_is_response_of_grn_first_ind(s.get_pop()));
+        assert(resp_worst != optima_is_response_of_grn_first_ind(s.get_pop()));
```

Comment:
same here, should use inputs instead of pop

Reply:
👍